CHANGES.md

```diff
@@ -15,6 +15,11 @@
   directory path in the scope of the `DatastoreManager` class.
 - [FIX] Corrected a case where two root nodes with identical revision IDs prevented selection of the
   correct new winning revision.
+- [FIX] Added migration on `Datastore` opening to repair datastores that have documents with
+  multiple identical revision IDs (as caused by issue 329). The migration will delete duplicate
```

Comment:
I wonder if we should explain that it might cause new revisions to be selected as the winner?